index.js

```diff
@@ -50,16 +50,21 @@ app.use(hbs.middleware({
 
 require("./routes");
 
-app.use(async(next) => {
+// Error handling middleware
+app.use(async (ctx,next) => {
 	try {
-		await next;
+		await next();
 	} catch (err) {
-		this.status = err.status || 500;
-		this.body = err.message;
-		this.app.emit("error", err, this);
+		ctx.status = err.status || 500;
+		ctx.render('error', {
```

Comment:
ESLint will require double quotes, so be sure to run `npm test` to make sure all your code adheres to it.